### DISCUSSION TITLE
Variety of Splash screens

### DISCUSSION THREAD

**Discussion by funclubdima** [airsdk/Adobe-Runtime-Support#3110] [Ideas]
I once saw how several teenagers entered the game and there was an “intro” by AIR from harman, but the thing is that the “intro” itself did not fit the style of the game, and I thought of suggesting that you add the ability to choose an intro so that There were several of them for different styles, and so that the intro could be included with a subscription to the AIR SDK

**Comment by shaucky** [comment]
I support in a way. Many of my current views on AIR come from my use of Unity. 
 
Leaving aside the fact that Unity seems to be planning to remove the personal version's start screen in the future version. In the current version, Unity allows developers to choose the color tone of the start screen, which I think is a great idea.
 
The start screen of Unity defaults to a black background and a white logo, but developers can also choose a color tone for the background and logo (for example, if the background turns dark red and the logo turns bright yellow).
 
Perhaps Harman can refer to this approach to provide more options for personal start screens.

**Reply by shaucky** [reply]
And also, the current Harman AIR start screen layout is also not aesthetically pleasing…

**Reply by funclubdima** [reply]
here it’s more about choosing styles and background colors, for example, 3D, flat and the current mode would be enough for me
